helm-charts/falcon-sensor/Chart.yaml

```diff
@@ -15,28 +15,35 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0
+version: 0.5.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 0.4.0
+appVersion: 0.5.0
 
 
 keywords:
-    - CrowdStrike
-    - Falcon
-    - EDR
-    - kubernetes
+  - CrowdStrike
+  - Falcon
+  - EDR
+  - kubernetes
+  - security
+  - monitoring
+  - alerting
 
 home: https://github.com/CrowdStrike/falcon-helm
 
 sources:
-    - https://github.com/CrowdStrike/falcon-helm
+  - https://github.com/CrowdStrike/falcon-helm
 
 maintainers:
-    - name: CrowdStrike Solution Architecture
-    - name: Shawn Wells
-      email: shawn.wells@crowdstrike.com
+  - name: CrowdStrike Solution Architecture
+  - name: Shawn Wells
+    email: shawn.wells@crowdstrike.com
 
-icon: https://github.com/CrowdStrike/falcon-helm/blob/main/images/crowdstrike-logo.svg
+icon: https://raw.githubusercontent.com/CrowdStrike/falcon-helm/main/images/crowdstrike-logo.svg
+
+annotations:
+  catalog.cattle.io/certified: partner
+  catalog.cattle.io/release-name: falcon-helm
```

Comment:
@shawndwells a thought here... we could add these annotations in when we submit to rancher rather than keep them added to this repo. It might make sense for handling other Kubernetes distros i.e. why have rancher annotations added to OpenShift... Thoughts?

Reply:
Would the content of the Helm chart differ? Not sure if Helm abstracts away the differences between k8s distros.

One immediate concern would be the administrative burden ("remember to re-add the tags!") vs being fully automated/syncd between the repos.

Reply:
Thinking about it... I am going to remove this for now. I don't want people thinking the upstream is the certified version vs what is in the partner repo. We can re-add it later if we decide that it is easier and better.